_maps_2024/10-youngsang-jun.md

```diff
@@ -0,0 +1,10 @@
+---
+day: 10
+creator: Jun, Youngsang
+title: My Childhood Map (Bupyeong-gu, Incheon, Korea)
+thumbnail: /assets/thumbnails/2024/10-youngsang-jun.jpg
+source: https://github.com/yjmark/30DayMapChallenge_2024_Jun/blob/main/Day10_PenandPaper/Day10_Jun.jpg
+---
+
+This paper-and-pen map shows the neighborhood around Buyeong-ro in Bupyeong-gu, Incheon, where I lived during my childhood. The places I remember most are those where spent time with my beloved family and friends, such as playgrounds, schools, and my friends' houses, especially the trampoline in front of the cathedral.
+![Screenshot of interactive map](assets/thumbnails/10-youngsang-jun.jpg)
```

Comment:
It looks like you're missing the year from this URL here. Same change for your other images.

```suggestion
![Screenshot of interactive map](assets/thumbnails/2024/10-youngsang-jun.jpg)
```